automation/jleechanorg_pr_automation/openai_automation/codex_cli_tasks.py

```
@@ -852,38 +882,83 @@ def create_pr_from_diff(self, task: dict, attempt: Optional[int] = None) -> dict
+                                "fetch",
+                                "origin",
+                                f"+{branch}:refs/remotes/origin/{branch}",
+                            ],
+                            capture_output=True,
+                            text=True,
+                            check=True,
+                            timeout=self.timeout,
+                        )
+                        self._checkout_detached_target(worktree_str, f"origin/{branch}")
+                        checked_out_detached = True
+                    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)